### PR TITLE
feat: Added unittest to test_views.py

### PR DESCRIPTION
**Pull request by MushtaqRossier**
**Description**: Added missing unittest from SONIC-123 and minor changes

**JIRA**: [SONIC-251](https://2u-internal.atlassian.net/browse/SONIC-251)

### CODE REVIEW

File: commerce_coordinator/settings/base.py [hunk 1]
```diff
@@ -349,9 +349,6 @@ def root(*path_fragments):
 ECOMMERCE_ADD_TO_BASKET_API_PATH = '/basket/add/'
 TITAN_URL = 'replace-me'
 
-# Enterprise URL
-ENTERPRISE_URL = 'SET-ME-PLEASE'
-
 # Timeout for enterprise client
 ENTERPRISE_CLIENT_TIMEOUT = os.environ.get('ENTERPRISE_CLIENT_TIMEOUT', 15)
 ENTERPRISE_USER_CACHE_TIMEOUT = 600  # Value is in seconds
```
Comment (aht007) [inline commerce_coordinator/settings/base.py:354]: Can we remove the unused `SYSTEM_ENTERPRISE_LEARNER_ROLE` and `ENTERPRISE_USER_CACHE_TIMEOUT`

**Review by aht007 — CHANGES_REQUESTED** [review]
Not a full review rn, will definitely take look later today. Requested a change.

File: commerce_coordinator/apps/lms/tests/test_views.py [hunk 2]
```diff
@@ -67,8 +69,26 @@ def test_view_rejects_unauthorized(self):
...
+        mock_ct_client.return_value.get_product_variant_by_course_run.side_effect = HTTPError('Error in CT search')
+        self.client.get(
+            self.url,
+            {'sku': ['sku1'], 'course_run_key': 'course-v1:MichiganX+InjuryPreventionX+1T2021'}
+        )
+        mock_ct_client.reset_mock(return_value=True, side_effect=True)
+        mock_logger.exception.assert_called_once_with(
+            '[get_product_variant_by_course_run] Failed to get CT course for course_run: '
+            'course-v1:MichiganX+InjuryPreventionX+1T2021 with exception: Error in CT search'
+        )
```
Comment (shafqatfarhan) [inline commerce_coordinator/apps/lms/tests/test_views.py:87]:
```suggestion
        mock_logger.exception.assert_called_once_with(
            '[get_product_variant_by_course_run] Failed to get CT course for course_run: '
            'course-v1:MichiganX+InjuryPreventionX+1T2021 with exception: Error in CT search'
        )
        mock_ct_client.reset_mock(return_value=True, side_effect=True)
```

File: commerce_coordinator/apps/lms/tests/test_views.py [hunk 3]
```diff
@@ -84,21 +104,20 @@ def test_run_rollout_pipeline_redirect_to_commercetools(self, is_redirect_mock):
                 'course-v1:MichiganX+InjuryPreventionX+1T2021'
             )
             is_redirect_mock.return_value = True
+            is_enterprise_mock.return_value = False
             self.client.force_authenticate(user=self.user)
             response = self.client.get(
                 self.url,
                 {'sku': ['sku1'], 'course_run_key': 'course-v1:MichiganX+InjuryPreventionX+1T2021'}
             )
+            # breakpoint()
```
Comment (shafqatfarhan) [inline commerce_coordinator/apps/lms/tests/test_views.py:113]: Please remove this comment.

**Review by shafqatfarhan — APPROVED** [review]
Approved with a couple of improvement comments.

**Review by aht007 — APPROVED** [review]
LGTM

**Pull request merged**